codegen/generated!.gotpl

```diff
@@ -84,9 +85,14 @@ func (e *executableSchema) Complexity(typeName, field string, childComplexity in
 	switch typeName + "." + field {
 	{{ range $object := .Objects }}
 		{{ if not $object.IsReserved }}
-			{{ range $field := $object.UniqueFields }}
+			{{ range $_, $fields := $object.UniqueFields }}
+			{{ $len := len $fields }}
+			{{ range $i, $field := $fields }}
+				{{ $ii := add $i 1 }}
```

Comment:
You know go can have multiple cases on the same line to avoid this fallthrough? eg

```go
	case "OverlappingFields.oneFoo", "OverlappingFields.twoFoo":
		... code ...
```

Should clean up both the generated code and the templates.

Reply:
Okay. I preferred one-field-one-line, but am happy to revise.

Reply:
@vektah Updated to avoid the `fallthrough`.